### DISCUSSION TITLE
Installer for Windows on ARM.

### DISCUSSION THREAD

**Discussion by moonman** [openscopeproject/TrguiNG#236] [Ideas]
Hi, would it be possible to build an installer for Windows 11 running on ARM devices (aarch64). There is more and more laptops available with Snapdragon X CPU and it would be nice to have a native app for them.

**Comment by qu1ck** [comment]
It should be possible in theory since tauri supports cross compilation for arm64 arch on x64.
But since I don't have an arm64 windows machine I can't test it myself and will need support for it in standard tooling. 
You can follow this issue if you are interested
https://github.com/tauri-apps/tauri-action/issues/952

**Reply by qu1ck** [reply]
You can get experimental arm64 build from the artifacts here [aarch64-pc-windows-msvc](https://github.com/openscopeproject/TrguiNG/actions/runs/11830088024/artifacts/2185635231)

**Reply by moonman** [reply]
Thanks very much! This works perfectly

**Comment by qu1ck** [comment]
Since artifacts can expire I'm uploading the same thing here.
[aarch64-pc-windows-msvc.zip](https://github.com/user-attachments/files/17823553/aarch64-pc-windows-msvc.zip)

Next release will include arm binary in normal location.
